src/image/upload.test.ts

```diff
@@ -64,40 +68,42 @@ describe('test file uploads', () => {
     `);
   });
 
-  test('upload one image via payload type, receive one response', async () => {
+  test('upload one image via bas64 payload type, receive one response', async () => {
     const accessToken = 'abc123';
     const client = new ImgurClient({ accessToken });
     const response = await upload(client, {
-      image: '/home/user/meme.jpg',
+      stream: createReadStream('/home/user/meme.jpg'),
       title: 'dank meme',
       description: 'the dankiest of dank memes',
     });
     expect(response).toMatchInlineSnapshot(`
       Object {
         "data": Object {
           "deletehash": "jyby9KJ",
-          "description": "the dankiest of dank memes",
+          "description": null,
```

Comment:
🤔

Reply:
Woops. It looks like I bungled a `jest -u` here and didn't resolve it?